readyplayerme/rpm-unity-sdk-core
# Release/v7.1.0

## [TICKETID](https://ready-player-me.atlassian.net/browse/TICKETID)

## Description

-  This release includes a rework to the shader override system to enable more flexibility with property mapping 






## How to Test

-  Install sdk from this release branch 
https://github.com/readyplayerme/rpm-unity-sdk-core.git#release/v7.1.0  
- The main change in functionality is around the shader overrides in the AvatarConfig 
![image](https://github.com/readyplayerme/rpm-unity-sdk-core/assets/7085672/aa44aecc-4066-46fb-8c2a-ac1452d4a34e)
- To test it is working you can try remapping some values and then loading an avatar with this config to confirm that it is being remapped correctly. 
- target shader property names can be found by inspecting the shader in Unity
If you need help figuring out what the property names should be I can help you out. 



## Checklist
